central/networkgraph/flow/datastore/internal/store/postgres/store.go

```diff
@@ -366,6 +377,10 @@ func (s *flowStoreImpl) readRows(rows pgx.Rows, pred func(*storage.NetworkFlowPr
 			flow.LastSeenTimestamp = protoconv.MustConvertTimeToTimestamp(*lastTime)
 		}
 
+		if updatedAt != nil {
+			flow.UpdatedAt = protoconv.MustConvertTimeToTimestamp(*updatedAt)
```

Comment:
`protoconv.MustConvertTimeToTimestamp` can panic. Is that ok in this case?

Reply:
We do the same for `LastSeen`. I think it's ok.